checkout/interfaces/controller/checkoutcontroller.go

```diff
@@ -319,7 +317,7 @@ func (cc *CheckoutController) SuccessAction(ctx context.Context, r *web.Request)
 }
 
 // ExpiredAction handles the expired cart action
-func (cc *CheckoutController) ExpiredAction(ctx context.Context, r *web.Request) web.Result {
+func (cc *CheckoutController) ExpiredAction(_ context.Context, _ *web.Request) web.Result {
```

Comment:
if all arguments are unused (`_`) you can omit them all together, such as `ExpiredAction(context.Context, *web.Request)`.